docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| napari term        | Description                                                                                                                                                                                                                                                                                                                                 |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| channel            | A channel is a grayscale image. In fluorescence microscopy,  each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue.                                                                                                                                                |
+| color map          | An array of colors used to map pixel data to the actual color values.                                                                                                                                                                                                                                                                       |
```

Comment:
How about: *"A colormap represents the value of each pixel as a particular color. There are many different types of colormaps available, see [link to matplotlib page]"*

* The matplotlib colormap documentation would be a good place to link to. https://matplotlib.org/stable/tutorials/colors/colormaps.html
* I think we typically use "colormap" as a single word. Could you double check by searching across the napari repo & tutorials repo? We'll need to make sure this is standardized (might need an issue opened to track this).

Reply:
Thanks for the suggestion, will update colormap definition.
- Do we use matplotlib colormap as the source for napari built-in colormaps? if that's the case we could make it more explicit. 
- Yes it's a mix in napari.org. Will use "colormap" for glossary and track the occasional "color map" separately.

Reply:
Yes, we do use the matplotlib colormaps, but not all of them.
There are maybe also be some non-matplotlib ones in there too? @alisterburt's first PR was about a better colormap for cryo-EM microscopy volumes (but maybe that was more of a vispy thing? I don't remember the details now)
If you need to know exactly what is included/omitted, you'd have to look here: https://github.com/napari/napari/tree/5f96d5d814aad697c367bdadbb1a57750e2114ad/napari/utils/colormaps

Reply:
Yeah we have some of the vispy colormaps, and some of the custom ones too. A dedicated docs page around our colormaps would be good too. @kevinyamauchi @jni do you remember if we ever did something like that? @andy-sweet was also cleaning up some of that code recently

Reply:
I'd be happy to create an issue and at least start the job of writing on some rough docs on colormaps and colors in napari, if only to better understand exactly what's currently supported. That could help motivate some follow-on work to simplify some of the associated code (e.g. achieving @tlambert03's [dream of using `pydantic.types.Color`](https://github.com/napari/napari/pull/3587#discussion_r746595476)).

Reply:
I created #3597 and assigned it to myself for now to track that work. I should be able to make some progress on rough docs soon, but anyone should free to assign themselves to the issue or comment on it.